grunt/tasks/react-dom.js

```diff
@@ -25,6 +25,8 @@ function build(name, filename) {
 }
 
 module.exports = function() {
+  /*
   build('ReactDOM', 'react-dom');
   build('ReactDOMServer', 'react-dom-server');
+  */
```

Comment:
this file should probably be deleted now that it’s built from source and not copying these over.